### ISSUE TITLE
Do I need signed ipsw with vieux?

### ISSUE THREAD

**Issue by yeahman45** [MatthewPierson/Vieux#82]
It still not clear for me whether a signed ipsw is required to be used with vieux

Can I upgrade an Ipad air 1st gen on IOS 7.1.2 to IOS 10.3.3 with an unsigned 10.3.3 ipsw?

**Comment by DanTheMann15** [comment]
Yes you can upgrade to iOS 10.3.3 using this tool without a pre-saved APTicket, because apple still signs that firmware OTA, except for the iPad Mini 2 (iPad4,6 model).

Just download the IPSW here: [iPad_64bit_10.3.3_14G60_Restore.ipsw](http://appldnld.apple.com/ios10.3.3/091-23378-20170719-CA983C78-6977-11E7-8922-3D9100BA0AE3/iPad_64bit_10.3.3_14G60_Restore.ipsw)
and then follow the instructions laid-out in the readme.

But before upgrading, i strongly recommend dumping your onboard 7.1.2 APTicket with it's generator, so it may allow you to go back to 7.1.2 when desired if somebody hopefully gets around the SEP compatibility limitations.

whatever you decide the do, have fun!

**Comment by MatthewPierson** [comment]
If @yeahman45 is running 7.1.2, then use [Déverser](https://github.com/MatthewPierson/deverser) to dump the blobs with it's valid generator, as @DanTheMann15 suggested.
